src/vstest.console/Processors/Utilities/ArgumemtProcessorUtilities.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors.Utilities
```

Comment:
should this also handle validation of arguments based on a list? instead of just returning any value from the raw string?

Reply:
We are handling the validation of argument here.